src/coreclr/clrdefinitions.cmake

```diff
@@ -256,6 +256,9 @@ function(set_target_definitions_to_custom_os_and_arch)
       target_compile_definitions(${TARGETDETAILS_TARGET} PRIVATE TARGET_APPLE)
       target_compile_definitions(${TARGETDETAILS_TARGET} PRIVATE TARGET_OSX)
     endif()
+    if (TARGETDETAILS_OS STREQUAL "unix_anyos")
```

Comment:
what exactly is `unix_anyos`? it represents both linux and osx? because there is also `unix_osx`?

Reply:
ah, its similar to how we have `universal` term...this is cross plat binary that can be used for altjit purpose on unix or macos I assume.

Reply:
Yes. Its not actually intended for altjit (although it does work for that case) its primary use is for crossgen2, which supports targeting all architecture/os pairs from any given 64bit build.